usecases/auth/authorization/conv/casbin_converter.go

```diff
@@ -78,7 +78,7 @@ func CasbinPolicies(casbinPolicies ...[][]string) (map[string][]authorization.Po
 	rolesPermissions := make(map[string][]authorization.Policy)
 	for _, p := range casbinPolicies {
 		for _, policyParts := range p {
-			name := policyParts[0]
+			name := strings.TrimPrefix(policyParts[0], "role:")
```

Comment:
Cna you use the constant? Or add a removeRolePrefix func?

Reply:
will do so, as a follow up there is contsant define but can't be used directly ,no need to have a function given in this package is not used in any other places